server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -136,155 +129,77 @@ def generate_summary_histogram():
     Return:
         Dash html div element containing request time to close histogram.
     """
-    histogram = html.Div(
-        # Histogram for the request timeToClose.
+    return html.Div(
         dcc.Graph(
             id="time_close_histogram", style={"height": "40vh",
                      "width": "48vw"}
         ), style=merge_dict(CHART_OUTLINE_STYLE, {"width": "48vw",
                  "height": "40vh"}))
-    return histogram
 
 
-def generate_council_name_dropdown():
+def generate_council_name_dropdown(output_id):
     """Generates the neighborhood council (nc) dropdown for the 
     comparison dashboard.
 
     This functions generates the html elements for the 
-    nc dropdown for the left side of the comparison dashboard.
+    nc dropdown for the comparison dashboard.
 
     Return:
         Dash html div element containing nc drop down for left pane filtering.
     """
-    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
-         " ", id="nc_comp_dropdown",
+    return html.Div(dcc.Dropdown(sorted(list(set(api_data_df["councilName"]))),
+         placeholder=" ", id=output_id,
                  placeholder="Select a Neighborhood Council..."),
                  style=merge_dict(INLINE_STYLE, {"width": "48.5vw"})),
-    return dropdown
-
-
-def generate_council_name_dropdown2():
-    """Generates the neighborhood council (nc) dropdown for the 
-    comparison dashboard.
-
-    This functions generates the html elements for the 
-    nc dropdown for the right side of the comparison dashboard.
 
-    Return:
-        Dash html div element containing nc drop down for right pane filtering.
-    """
-    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
-         " ", id="nc_comp_dropdown2",
-                 placeholder="Select a Neighborhood Council..."),
-                 style=merge_dict(INLINE_STYLE, {"width": "48.5vw"}))
-    return dropdown
-
-
-def generate_comparison_total_req():
+def generate_comparison_total_req(output_id):
     """Generates the indicator visual for the 
     total number of requests.
 
     This functions generates the html elements for the 
-    total number of requests for the left side of the 
-    comparison dashboard.
+    indicator visual with matching output_id showing the total number of 
+    requests for the comparison dashboard.
 
```

Comment:
The new arg is not being used in this function.

Also, please add `Args` section

Reply:
Thanks for the catch - resolved.